src/controllers/LevelController.js

```diff
@@ -27,7 +24,7 @@ class LevelController {
         level: user.level.value
       })
 
-      if (user.level.value > previousLevel) {
+      if (user.level.value > previousLevel && providers.length > 0) {
```

Comment:
```suggestion
      if (user.level.value > previousLevel) {
```

@marianaviana acredito que você não precisa dessa condição, pois na linha 28 um array vazio simplesmente não terá loop